windfile.py

```diff
@@ -37,57 +39,37 @@ def get(self, lat, lon, altitude, time):
         return u, v
 
     def get_indices(self, lat, lon, alt, time):
-        lat = 90 - lat
-        lon = lon % 360
+        lat = (90 - lat) * self.resolution_lat_multiplier
+        lon = (lon % 360) * self.resolution_lon_multiplier
 
         time = (time - self.time)/self.interval
         pressure = self.get_pressure_index(alt)
         
-        print("bounds:", lat, lon, pressure, time)
         return lat, lon, pressure, time
 
     def get_pressure_index(self, alt):
-
-        # IGNORE: code for scipy interpolation
-        #x = np.arange(range(len(self.levels)))
-        #y = self.levels
-        #f = interpolate.interp1d(y, x)
-
+        indices = np.arange(0, len(self.levels))
+        interp_function = interpolate.interp1d(self.levels, indices, bounds_error=False, fill_value=(0, len(self.levels)-1))
```

Comment:
Okay, great that this works! Just FYI I think you can pass `assume_sorted=True` to save some compute, and also make this entire `interp_function` in the constructor.